projects/plugins/crm/includes/ZeroBSCRM.AdminPages.php

```diff
@@ -269,106 +267,72 @@ function zeroBSCRM_pages_admin_team() {
-		<i class="add icon"></i> 
-			<?php esc_html_e( 'Add New Team Member', 'zero-bs-crm' ); ?>
+		<a class="ui button black" href="<?php echo esc_url( admin_url( 'user-new.php?zbsslug=zbs-add-user' ) ); ?>">
+			<i class="add icon"></i>
+			<?php esc_html_e( 'Add new team member', 'zero-bs-crm' ); ?>
 		</a>
 
-	</div>
-
-	<div class='clear'></div>
+		<script type="text/javascript">
+			document.getElementById('jpcrm-search-wp-users').addEventListener('click', function() {
+				document.getElementById('jpcrm-users-search').submit();
+			});
+		</script>
 
-	<div class="ui divider"></div>
+	</div>
 
-	<table class="ui fixed single line celled table" id="zbs-team-user-table">
-		<tbody>
-		<th style="width:40px;"><?php esc_html_e( 'ID', 'zero-bs-crm' ); ?></th>
-		<th><?php esc_html_e( 'Team member', 'zero-bs-crm' ); ?></th>
-		<th><?php esc_html_e( 'Role', 'zero-bs-crm' ); ?></th>
-		<th><?php esc_html_e( 'Last login', 'zero-bs-crm' ); ?></th>
-		<th><?php esc_html_e( 'Manage permissions', 'zero-bs-crm' ); ?></th>
+	<div id="jpcrm-team-user-table">
+		<div class="fit-col"><?php esc_html_e( 'ID', 'zero-bs-crm' ); ?></div>
+		<div><?php esc_html_e( 'Team member', 'zero-bs-crm' ); ?></div>
+		<div><?php esc_html_e( 'Role', 'zero-bs-crm' ); ?></div>
+		<div><?php esc_html_e( 'Last login', 'zero-bs-crm' ); ?></div>
+		<div><?php esc_html_e( 'Manage permissions', 'zero-bs-crm' ); ?></div>
 		<?php
-		foreach ( $zbsRoleIDs as $ID ) {
-			$user = get_user_by( 'ID', $ID );
-
-			// zbs_prettyprint($user);
+		foreach ( $crm_role_ids as $id ) {
+			$user = get_user_by( 'ID', $id );
 
-			$edit_url = admin_url( 'user-edit.php?user_id=' . $ID . '&zbsslug=zbs-edit-user' );
+			$edit_url = admin_url( 'user-edit.php?user_id=' . $id . '&zbsslug=zbs-edit-user' );
 
 			$caps_output = '';
 			foreach ( $user->caps as $k => $v ) {
 				$caps_output .= ' ' . zeroBSCRM_caps_to_nicename( $k );
 			}
-
-			echo '<tr><td>' . esc_html( $ID ) . '</td><td>' . jpcrm_get_avatar( $ID, 30 ) . "<div class='dn'>" . esc_html( $user->display_name ) . '</div></td><td>' . esc_html( $caps_output ) . '</td>';
-
-			echo '<td>' . esc_html( zeroBSCRM_wpb_lastlogin( $ID ) . ' ' . __( 'ago', 'zero-bs-crm' ) ) . '</td>';
-
-			echo "<td><a href='" . esc_url( $edit_url ) . "'' data-uid='" . esc_attr( $ID ) . "' class='zbs-perm-edit ui button mini black'>"; // phpcs:ignore WordPress.NamingConventions.ValidVariableName.VariableNotSnakeCase
-
-			esc_html_e( 'Manage permissions', 'zero-bs-crm' );
-
-			echo '</a></td>';
-
-			echo '</tr>';
-
-			// zbs_prettyprint($user);
+			?>
+			<div class="fit-col"><?php echo esc_html( $id ); ?></div>
+			<div><?php echo jpcrm_get_avatar( $id, 30 ); ?><?php echo esc_html( $user->display_name ); ?></div>
+			<div><?php echo esc_html( $caps_output ); ?></div>
+			<div><?php echo esc_html( zeroBSCRM_wpb_lastlogin( $id ) . ' ' . __( 'ago', 'zero-bs-crm' ) ); ?></div>
```

Comment:
I've not tested this, but would the following make more sense?
```suggestion
			<div><?php printf( esc_html__( '%s ago', 'zero-bs-crm' ), esc_html( zeroBSCRM_wpb_lastlogin( $id ) ) ); ?></div>
```

Reply:
Indeed, I much prefer a translation string with more context. Your version works, but I did it slightly differently in f6272fd439.